scripts/generate-integration-pages.ts

```diff
@@ -38,12 +38,21 @@ class IntegrationPagesBuilder {
 	readonly #sourceBranch: string;
 	readonly #sourceRepo: string;
 	readonly #deprecatedIntegrations = new Set(['turbolinks']);
+	readonly #nonCoreIntegrations: Map<string, { packageName: string; repo: string; }> = new Map();
 	readonly #i18nNotReadyIntegrations = new Set<string>([]);
 
 	constructor(opts: { githubToken?: string; sourceBranch: string; sourceRepo: string }) {
 		this.#githubToken = opts.githubToken;
 		this.#sourceBranch = opts.sourceBranch;
 		this.#sourceRepo = opts.sourceRepo;
+		this.#nonCoreIntegrations.set('netlify', {
+			packageName: '@astrojs/netlify',
+			repo: 'withastro/netlify-adapter'
+		});
```

Comment:
Chatted a bit with @sarah11918 earlier today and agreed that any of these that remain “official” (i.e. under the `withastro` GitHub org and `@astrojs` npm scope) make sense to keep in docs. Not sure where all of this will land, but if some adapters end up with that status (like the Netlify one in its new location), then something like this change from the initial commit probably will make sense.